scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+'''Tool to import browser history data from other browsers. Although, safari
+support is still on the way.'''
+
+
+import argparse
+import sqlite3
+import sys
+
+
+def parser():
+    """Parse command line arguments."""
+    description = 'This program is meant to extract browser history from your'\
+                  'previous browser and import them into qutebrowser.'
+    epilog = 'Databases:\n\tQute: Is named "history.sqlite" and can be found '\
+             'at your --basedir. In order to find where your basedir is you '\
+             'can run ":open qute:version" inside qutebrowser.'\
+             '\n\tFirerox: Is named "places.sqlite", and can be found at your'\
+             'system\'s profile folder. Check this link for where it is locat'\
+             'ed: http://kb.mozillazine.org/Profile_folder'\
+             '\n\tChrome: Is named "History", and can be found at the respec'\
+             'tive User Data Directory. Check this link for where it is locat'\
+             'ed: https://chromium.googlesource.com/chromium/src/+/master/'\
+             'docs/user_data_dir.md\n\n'\
+             'Example: $this_script.py -b firefox -s /Firefox/Profile/places.'\
+             'sqlite -d /qutebrowser/data/history.sqlite'
+    parsed = argparse.ArgumentParser(
+        description=description, epilog=epilog,
+        formatter_class=argparse.RawTextHelpFormatter
+    )
+    parsed.add_argument('-b', '--browser', dest='browser', required=True,
+                        type=str, help='Browsers: {firefox, chrome, safari}')
+    parsed.add_argument('-s', '--source', dest='source', required=True,
+                        type=str, help='Source: fullpath to the sqlite data'
+                        'base file from the source browser.')
+    parsed.add_argument('-d', '--dest', dest='dest', required=True, type=str,
+                        help='Destination: The fullpath to the qutebrowser '
+                        'sqlite database')
+    return parsed.parse_args()
+
+
+def open_db(data_base):
+    """Open connection with database."""
+    try:
+        conn = sqlite3.connect(data_base)
+        return conn
+    except Exception as any_e:
+        print('Error: {}'.format(any_e))
+        raise('Error: There was some error trying to to connect with the [{}]'
+              'database. Verify if the filepath is correct or is being used.'.
+              format(data_base))
```

Comment:
Wait, does raising a string actually work with Python 3? Anyways, I wonder if it's worth catching the exception here at all, as it just hides some information.